src/components/Tool/Appeal/List/AppealsListFilterPanel/AppealsListFilterPanel.tsx

```diff
@@ -142,6 +185,13 @@ export const AppealsListFilterPanel: React.FC<FilterPanelProps & BoxProps> = ({
                   <Close titleAccess={t('Close')} />
                 </IconButton>
               </Box>
+              <LinkButton
+                disabled={noActiveFilters}
+                onClick={handleClearAllClick}
+                variant="outlined"
+              >
+                {t('Clear All')}
```

Comment:
I don't think this button is needed. It is not on the angular.

Reply:
I know it's not on Angular, but when using it, I like to see all the contacts and be able to clear filters easily. Especially since we push people into the `asked` filter on load.

Reply:
When I've used appeals, I've never thought of them as filters. I've thought of them more as categories. In Angular there is no `Clear` button and there is _**no**_ option to **_not_** have a category/filter selected. The default is always the `Asked` category. There really is no need to see all the contacts at once. So I would suggest keeping the angular functionality.